utils/build/clean.js

```diff
@@ -1,6 +1,6 @@
-const { packages } = require("../list_packages");
```

Comment:
I did not know this exists, do we need this script at all - where do we use it?

Reply:
It *should be used* in `npm run clean`, but there's a shortcut instead. We might follow-up with updating these build scripts